saleor/graphql/app/utils.py

```diff
@@ -40,3 +40,8 @@ def validate_app_is_not_removed(
                 )
             }
         )
+
+
+class CircuitBreakerState:
```

Comment:
Enums or enum-like classes are usually stored in the module's main `__init__` file, so I suggest `saleor/app/__init__.py` for this one.